app/src/main/java/com/fstyle/androidtrainning/screen/main/fragments/online/subfragment/topalbum/TopAlbumFragment.java

```diff
@@ -0,0 +1,36 @@
+package com.fstyle.androidtrainning.screen.main.fragments.online.subfragment.topalbum;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.fstyle.androidtrainning.R;
+import com.fstyle.androidtrainning.screen.main.fragments.mysong.subfragment.favoritesong
```

Comment:
ERROR: Unused import: com.fstyle.androidtrainning.screen.main.fragments.mysong.subfragment.favoritesong.FavoriteSongPresenter. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports